include/odp/api/spec/ipsec.h

```diff
@@ -991,6 +991,21 @@ typedef struct odp_ipsec_out_opt_t {
 			/** Use fragmentation mode option */
 			uint32_t frag_mode: 1;
 
+			/** Use IP parameters option */
+			uint32_t ip_param:  1;
+
+			/** Use TFC padding length option */
+			uint32_t tfc_pad:   1;
+
+			/** Tunnel mode TFC dummy packet. This can be used only
```

Comment:
Limiting this option to tunnel mode only seems logical but not that effective for an application. Maybe we can allow using this option together with L3 header override to create transport mode dummy packets. Then it would be just easier to inject dummy packets in transport case.